indico/modules/rb/templates/emails/reservations/confirmation_email_to_user.txt

```diff
@@ -4,6 +4,11 @@
 {% block state -%}
 Your booking has been ACCEPTED.
 This is the final confirmation.
+{% if reason %}
```

Comment:
Matter of style, but if you use `{%- if reason %}` you can add a linebreak before this line to keep things a bit more separate in the template. Jinja removes any whitespace on the side where the `-` is.